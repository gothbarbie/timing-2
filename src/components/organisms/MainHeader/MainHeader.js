import React from 'react'
import styled from 'styled-components'

const MainHeaderStyle = styled.header`
  display: flex;
  justify-content: center;
  background-color: ${({ theme }) => theme.colors.primary};
  color: ${({ theme }) => theme.colors.white};
  padding: 1rem 0;
`

const MaxWidth = styled.div`
  display: flex;
  align-items: center;
  justify-content: space-between;
  width: 100%;
  max-width: 1200px;
  padding: 0 1rem;
`

const Logo = styled.h1`
  font-size: 2.8rem;
  font-weight: 500;
  margin: 0;
`

const Username = styled.div`
  font-size: 1.6rem;
  font-weight: 400;
  margin: 0;
`

const MainHeader = () => (
  <MainHeaderStyle>
    <MaxWidth>
      <Logo>Timing</Logo> <Username>Username</Username>
    </MaxWidth>
  </MainHeaderStyle>
)

export default MainHeader
