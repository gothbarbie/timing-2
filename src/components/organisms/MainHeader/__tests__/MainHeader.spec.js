import React from 'react'
import { shallow } from 'enzyme'

import MainHeader from '../MainHeader'

describe('organisms/MainHeader', () => {
  it('renders', () => {
    const component = shallow(<MainHeader />)

    expect(component).toMatchSnapshot()
  })

})
