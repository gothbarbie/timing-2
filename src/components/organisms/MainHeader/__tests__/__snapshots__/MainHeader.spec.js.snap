// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`organisms/MainHeader renders 1`] = `
<MainHeaderStyle>
  <MaxWidth>
    <Logo>
      Timing
    </Logo>
     
    <Username>
      Username
    </Username>
  </MaxWidth>
</MainHeaderStyle>
`;
